cmd/starbridge/integrations/ethereum.go

```diff
@@ -47,41 +48,60 @@ func Ethereum2Transaction(conn *ethclient.Client, txReceipt *types.Receipt, tx *
 	if !IsMyContractAddress(tx.To().Hex()) {
 		return nil, fmt.Errorf("unsupported receiver address '%s'", tx.To().Hex())
 	}
-
 	fromAddress, e := getFromAddress(tx)
```

Comment:
💡 It's common to name errors `err`. I think this is the first time I've seen `e` in Go code. It helps others read the code and see the errors at a glance if this code looks like other Go code.
```suggestion
	fromAddress, e := getFromAddress(tx)
```

Reply:
I have a habit of using `e` since it's shorter. Will transition to using `err` since that is standardized

Reply:
fixed in next PR: https://github.com/stellar/starbridge/pull/29